### PR TITLE
Add Business Insurance

### PR DESCRIPTION
**Pull request by conatus**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I've checked the spec (e.g. Figma file) and documented any divergences.
- [ ] My code follows the code style of this project.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I've updated the documentation accordingly.
- [ ] Replace unused checkboxes with bullet points.

### CODE REVIEW

File: business-insurance.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+We have insurance coverage from **Finsbury**, brokered by **[Simply Business](simplybusiness.co.uk)**, since March 12, 2021.
+
+# Policy details
+
+Policy start date: 12/03/2021, up for renewal March 11, 2024.
+
+Policy number: **FHGP3036588XB**
```
Comment (conatus) [inline business-insurance.md:7]: Is it problematic to have this in the public domain? I think not but will briefly check.

File: business-insurance.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    
+    > £100,000
+
+- Excess protection
+    
+    > £5,000
+    
+- Office and Portable equipment
+    
+    Up to £10,000. If you have a laptop that is owned by Common Knowledge it is covered under this policy.
```
Comment (conatus) [inline business-insurance.md:60]: I added this in.

**Comment by conatus** [comment]
- [ ] Where are the policy documents?
- [ ] Is it okay to have the policy number in the public domain?

**Review by GemCopeland — APPROVED** [review]

**Comment by GemCopeland** [comment]
@conatus Policy documents are accessible via their website. IMO it's ok to have the policy number publicly, but we could also just remove it and refer people to Notion.

**Pull request closed without merging**